scania-digital-design-system/sdds-website
# Feature/bump version

**Describe pull-request**  
_Describe what the pull-request is about_
Update content and bump versions

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#921](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/921)

**How to test**  
_Add description how to test if possible_
Run local or start the preview in netlify
Check badge for example
